pkg/controller/user/delete.go

```diff
@@ -59,7 +59,7 @@ func (c *Controller) HandleDelete() http.Handler {
 			return
 		}
 
-		flash.Alert("Successfully removed user '%v from realm'", user.Email)
+		flash.Alert("Successfully removed user '%v' from realm", user.Email)
```

Comment:
Bad rebase?

Reply:
Actually, looks like that quote shouldn't be anywhere in this string.

Reply:
Sorry this is kind of unrelated, just saw the message at one point while testing and didn't like the weird quotes